pkgs/tools/package-management/nix-doc/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, rustPlatform, fetchFromGitHub, pkgs, ... }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "nix-doc";
+  version = "0.3.3";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "lf-";
+    repo = "nix-doc";
+    sha256 = "0vd7159y5w8jjgaw51kfr3z3r50299gvw7vjchpqx3nwmdink8bh";
+  };
+
+  doCheck = true;
+  buildInputs = with pkgs; [ boost nix ];
+
+  nativeBuildInputs = with pkgs; [ pkg-config ];
```

`nativeBuildInputs` are usually sorted above regular `buildInputs`, though I'm not sure if theres any strict guidelines on this.

```suggestion
  nativeBuildInputs = [ pkg-config ];
```